src/components/StakingTimeline/index.tsx

```diff
@@ -107,10 +104,8 @@ export const LegacyStakesDepositSteps: FC = () => {
         >
           You will need to run a node for applications that you have authorized
           to earn rewards. If you don’t have one, learn how to do it{" "}
-          <InternalLink to={APPLICATION_DOCS_URL}>here</InternalLink> or contact{" "}
-          <InternalLink to={STAKING_PROVIDER_URL}>
-            a Staking Provider
-          </InternalLink>
+          <Link to={APPLICATION_DOCS_URL}>here</Link> or contact{" "}
+          <Link to={STAKING_PROVIDER_URL}>a Staking Provider</Link>
```

Comment:
Non-blocking:

Sam as above:

```suggestion
          <Link to={STAKING_PROVIDER_URL}>a Staking Provider</Link>.
```